app/styles/app.scss

```diff
@@ -106,4 +108,31 @@ picture * {
 
 .inbox {
   height: 600px;
+}
+
+.csv_leaderboard {
```

Comment:
Is all this css really necessary?
If not try to stick with bootstrap's table or refer similar structure in motley